build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.3.70'
+    ext.kotlin_version = '1.5.10'
```

Comment:
Kotlin 1.5 support `sealed interface` so one task can be simplified:
The `Classes/Sealed classes` task should consist of replacing
`interface Expr` by `sealed interface Expr`
In `Classes/Sealed classes/src/Task.kt`
The code:
```
sealed class Expr
class Num(val value: Int) : Expr()
class Sum(val left: Expr, val right: Expr) : Expr()
```
Could be:
```
sealed interface Expr
class Num(val value: Int) : Expr
class Sum(val left: Expr, val right: Expr) : Expr
```

Reply:
That's a good point! Thanks, fixed